rmgpy/rmg/input.py

```diff
@@ -206,6 +206,20 @@ def simple_reactor(temperature,
             elif value[1] < value[0]:
                 raise InputError('Initial mole fraction range out of order: {0}'.format(key))
 
+    # normalize mole fractions if applicable
+    totalInitialMoles = sum(initialMoleFractions.values())
```

Comment:
This should be `snake_case`. Also, this doesn't seem to account for mole fraction ranges.